src/main/java/com/braintreepayments/apollo_tracing_uploader/impl/Uploader.java

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.apollo_tracing_uploader.impl;
```

Comment:
Why did you move this to the `impl` package? This is meant for concrete implementations of interfaces.